ultralist/app.go

```diff
@@ -236,7 +236,6 @@ func (a *App) ExpandTodo(input string) {
 		a.AddTodo(strings.Join(args, ""))
 	}
 
-	a.TodoList.Delete(id)
```

Comment:
I don't understand why this was here. It was always deleting one of the new todos.

Reply:
The original intention of expanding a todo was to create many todos out of a single one.  I think we should talk about this change in a separate PR.

Reply:
Removed.